src/helpers/folder_helpers.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# Helper methods for dealing with directories.
+
+
+# Changes into the given directory
+function change_directory {
+  local directory=$1
+  pwd
+  run_git_command "cd $directory"
+  pwd
```

Comment:
Sorry, debugging leftovers. This was pretty tricky to test, because of disappearing folders in the middle of the spec, which left Ruby dumbfounded.